docs/readthedocs/source/doc/Chronos/Howto/how_to_use_benchmark_tool.md

```diff
@@ -0,0 +1,135 @@
+# Use Chronos benchmark tool
+This page demonstrates how to use Chronos benchmark tool to benchmark forecasting performance on platforms.
+
+## Basic Usage
+The benchmark tool is installed automatically when Chronos is installed using conda. To get preliminary knowledge of forecasting performance on platform, run benchmark tool with default options using following command:
+```bash
+benchmark-chronos -l 96 -o 720
+```
+```eval_rst
+.. note::
+    **Required Options**:
+
+     **-l/--lookback** and **-o/--horizon** are required options for Chronos benchmark tool. Use **-l/--lookback** to specify the history time steps while use **-o/--horizon** to specify the output time steps.
+```
+By default, the tool will load `tsinghua_electricity` dataset and train a `TCNForecaster` with input lookback and horizon parameters under `pytroch` framework. As it loads, it prints information about hardware, Nano environment variables and benchmark parameters. When benchmarking is completed, it reports the average throughput during training process. Users may be able to improve forecasting performance by following suggested changes on Nano environment variables.
+
+Besides the default usage, more execution parameters can be set to obtain more benchmark results. Read on to learn more about the configuration options available in Chronos benchmark tool.
+
+## Configuration Options
+The benchmark tool provides various options for configuring execution parameters. Some key configuration options are introduced in this part and a list of all options is given in [**Advanced Options**](https://bigdl.readthedocs.io/en/latest/doc/Chronos/Howto/how_to_use_benchmark_tool.html#advanced-options).
+
+### Model
+The tool provides several built-in time series forecasting models, including TCN, LSTM, Seq2Seq, NBeats and Autoformer. To specify which model to use, run benchmark tool with `-m/--model`. If not specified, TCN is used as the default.
+```bash
+benchmark-chronos -m lstm -l 96 -o 720
+```
+
+### Stage
+Regarding a model, training and inference stages are most concerned. By setting `-s/--stage` parameter, users can obtain knowledge of throughput during training (`-s train`), throughput during inference (`-s throughput`) and latency of inference (`-s latency`). If not specified, train is used as the default.
+```bash
+benchmark-chronos -s latency -l 96 -o 720
+``` 
+
+### Dataset
+Several built-in datasets can be chosen, including nyc_taxi and tsinghua_electricity. If users are with poor Internet connection and hard to download dataset, run benchmark tool with `-d synthetic_dataset` to use synthetic dataset. Default to be tsinghua_electricity if `-d/--dataset` parameter is not specified.
```

Comment:
Is it necessary to illustrate that user needs to manually download `synthetic_dataset` and put into the specific path?

Reply:
we don't need to download synthetic dataset right, that's something we need to do for electricity.

Maybe we could change the default dataset to synthetic dataset rather than electricity in future PR.

Reply:
well, I add a note box here to remind users of downloading electricity dataset. If we change the default dataset to synthetic dataset, guide will be updated together.